local-cli/upgrade/upgrade.js

```diff
@@ -15,7 +15,7 @@ const Promise = require('promise');
 const yeoman = require('yeoman-environment');
 const semver = require('semver');
 
-module.exports = function upgrade(args, config) {
+function upgrade(args, config) {
   args = args || process.argv;
```

Comment:
Do we need this line? I don't think it's needed. Commander will always take process.argv here.